### PR TITLE
Added string filtering for "madrich"

### PR DESCRIPTION
**Pull request by alongalky**

### CODE REVIEW

File: scripts/facebook-scanner.js [hunk 1]
```diff
@@ -4,13 +4,33 @@
...
 
-makePostsRed()
+/*
+    This function returns true if the DOM element or its children
+    contains the string "מדריך"
+*/
+var noMadrichPredicate = function(element) {
+    var string = element.outerHTML
+    var substrings = [
+      'מדריך',
```
Comment (pinkasey) [inline scripts/facebook-scanner.js:25]: '[מדריך]',
'[מדריכה]',
'[מדריכ/ה]',
'[מדריך/ה]',
[Mentor]'

Case insensitive

**Review by pinkasey — CHANGES_REQUESTED** [review]

**Comment by pinkasey** [comment]
Thanks for the contribution!
Please see my comment.
If you dont have time to make that change - i'll accept this PR and do it myself later.

**Comment by alongalky** [comment]
If you're eager you can go ahead, if not I'll probably get to it in the next few days

**Comment by pinkasey** [comment]
Thanks ☺
No rush

**Review by pinkasey — APPROVED** [review]

**Pull request merged**